rnd/autogpt_server/autogpt_server/app.py

```diff
@@ -19,10 +27,14 @@ def run_processes(processes: list[AppProcess], **kwargs):
 
 
 def main(**kwargs):
+    settings = get_config_and_secrets()
+    set_start_method("spawn", force=True)
+    freeze_support()
```

Comment:
These has been done on `process.py`

Reply:
i can do on both, but it was crashing when frozen if not here

Reply:
okay so if i set the spawn method here but not freeze, it works???